### PR TITLE
Improve the layout of the add/edit metric screen

### PR DESCRIPTION
**Pull request by kharbutli-meta**
## Description

GTT-1898: Change the layout of the add/edit metric screen to make it more user-friendly.

## Testing

* Unit tests.
* Local run.

Describe how you tested your changes and/or give instructions to the reviewers on how they can verify them.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "react-waypoint": "^10.1.0",
     "recharts": "^2.0.4",
     "typescript": "~3.7.2",
-    "uswds": "^2.11.2",
+    "uswds": "^2.13.3",
```
Comment (mabreuortega) [inline frontend/package.json]: red hot danger here!!!

**Review by mabreuortega — COMMENTED** [review]
I'm really concerned with the upgrade of USWDS

In addition, the is some css off in the Add/Edit metric check boxes. The background is clear

image

File: frontend/package.json (hunk 1 above)
Comment (kharbutli-meta) [reply frontend/package.json]: Fixed both

**Review by mabreuortega — APPROVED** [review]
LGTM

**Pull request merged**